src/services/telementryService.js

```diff
@@ -56,43 +56,54 @@ export const initialize = ({ context, config, metadata }) => {
 };
 
 export const start = duration => {
-  CsTelemetryModule.instance.telemetryService.raiseStartTelemetry({
-    options: getEventOptions(),
-    edata: {
-      type: 'content',
-      mode: 'play',
-      stageid: url,
-      duration: Number((duration / 1e3).toFixed(2)),
-    },
-  });
+  if (process.env.REACT_APP_TELEMETRY_MODE === 'ET' || process.env.REACT_APP_TELEMETRY_MODE === 'NT' || process.env.REACT_APP_TELEMETRY_MODE === 'DT') {
+    CsTelemetryModule.instance.telemetryService.raiseStartTelemetry({
+      options: getEventOptions(),
+      edata: {
+        type: 'content',
+        mode: 'play',
+        stageid: url,
+        duration: Number((duration / 1e3).toFixed(2)),
+      },
+    });
+  }
+
 };
 
 export const response = (context, options) => {
-  CsTelemetryModule.instance.telemetryService.raiseResponseTelemetry(
-    {
-      ...context,
-    },
-    getEventOptions()
-  );
+  if (process.env.REACT_APP_TELEMETRY_MODE === 'ET' || process.env.REACT_APP_TELEMETRY_MODE === 'NT' || process.env.REACT_APP_TELEMETRY_MODE === 'DT') {
+    CsTelemetryModule.instance.telemetryService.raiseResponseTelemetry(
+      {
+        ...context,
+      },
+      getEventOptions()
+    );
+  }
+
 };
 
 export const end = () => {
-  CsTelemetryModule.instance.telemetryService.raiseEndTelemetry({
-    edata: {
-      type: 'content',
-      mode: 'play',
-      pageid: url,
-      summary: [],
-      duration: '000',
-    },
-  });
+  if (process.env.REACT_APP_TELEMETRY_MODE === 'ET' || process.env.REACT_APP_TELEMETRY_MODE === 'NT' || process.env.REACT_APP_TELEMETRY_MODE === 'DT') {
+    CsTelemetryModule.instance.telemetryService.raiseEndTelemetry({
+      edata: {
+        type: 'content',
+        mode: 'play',
+        pageid: url,
+        summary: [],
+        duration: '000',
+      },
+    });
+  }
+
 };
 
-export const interact = id => {
-  CsTelemetryModule.instance.telemetryService.raiseInteractTelemetry({
-    options: getEventOptions(),
-    edata: { type: 'TOUCH', subtype: '', id, pageid: url },
-  });
+export const interact = (telemetryMode,id) => {
```

Comment:
The 'id' should be the first parameter that is required, while optional parameters should be placed at the end in a functional declaration.